src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -43,99 +19,17 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
-  });
-
   const venueTabId = "chat-sidebar-tab-venue";
   const privateTabId = "chat-sidebar-tab-private";
 
   return (
     <div
       role="dialog"
       aria-labelledby={isVenueChat ? venueTabId : privateTabId}
-      className={containerStyles}
+      className={styles.chatSidebar}
     >
-      {isExpanded ? (
-        <>
-          <div className="chat-sidebar__header">
-            <button
-              aria-label="Hide chat"
-              className="chat-sidebar__controller"
-              onClick={toggleSidebar}
-            >
-              <FontAwesomeIcon icon={faChevronRight} size="sm" />
-            </button>
-            <div className="chat-sidebar__tabs" role="tablist">
-              <button
-                role="tab"
-                id={venueTabId}
-                aria-label="Chat"
-                aria-selected={isVenueChat}
-                className={venueChatTabStyles}
-                onClick={selectVenueChat}
-              >
-                Chat
-              </button>
-              <button
-                role="tab"
-                id={privateTabId}
-                aria-label={privateChatTabTitle}
-                aria-selected={isPrivateChat}
-                className={privateChatTabStyles}
-                onClick={selectPrivateChat}
-              >
-                {privateChatTabTitle}
-              </button>
-            </div>
-          </div>
-          <div role="tabpanel" className="chat-sidebar__tab-content">
-            {isVenueChat && <VenueChat space={venue} />}
-            {isPrivateChat && <PrivateChats recipient={recipient} />}
-          </div>
-        </>
-      ) : (
-        <div className="chat-sidebar__header">
-          <button
-            aria-label="Show chat"
-            className="chat-sidebar__controller"
-            onClick={toggleSidebar}
-          >
-            <>
-              <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <div className="chat-sidebar__controller-wrapper">
-                <FontAwesomeIcon icon={faCommentDots} size="lg" />
-                <span className="chat-sidebar__controller-text">Chat</span>
-              </div>
-            </>
-          </button>
-          <button
-            aria-label="Show chat"
-            className="chat-sidebar__controller chat-sidebar__private-chat"
-            onClick={togglePrivateChatSidebar}
-          >
-            <>
-              <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <div className="chat-sidebar__controller-wrapper">
-                <FontAwesomeIcon icon={faEnvelope} size="lg" />
-                <span className="chat-sidebar__controller-text">Messages</span>
-                {newPrivateMessageRecived > 0 && (
-                  <div className="chat-sidebar__controller--new-message"></div>
-                )}
-              </div>
-            </>
-          </button>
-        </div>
-      )}
+      {isVenueChat && <VenueChat />}
```

Comment:
`SpaceChat` or `PublicChat` ?

Reply:
I didn't want to tackle renaming this in this PR. I can see us restructuring the chat bar completely when we have a better view on the spec for it.